src/EditorFeatures/CSharpTest/Completion/CompletionProviders/OverrideCompletionProviderTests.cs

```diff
@@ -3042,5 +3042,25 @@ public override bool Bar(Baz baz)
 
             await VerifyCustomCommitProviderAsync(markupBeforeCommit, "Bar(NS2.Baz baz)", expectedCodeAfterCommit);
         }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
```

Comment:
nit: WorkItem for #47941

Reply:
i wasn't aware there was an issue.  this came out as part of my records test pass.  if this fails, i can add that workitem and resubmit.  if it passes though, it's fine to not have the workitem and we can just close the issue as fixed.